tests/foreman/ui/test_rhc.py

```diff
@@ -65,16 +65,18 @@ def module_rhc_org(module_target_sat):
 
 
 @pytest.fixture()
-def fixture_setup_rhc_satellite(request, module_target_sat, module_rhc_org):
+def fixture_setup_rhc_satellite(
+    request,
+    module_target_sat,
+    module_rhc_org,
+    module_entitlement_manifest,
+):
     """Create Organization and activation key after successful test execution"""
     yield
     if request.node.rep_call.passed:
         if settings.rh_cloud.crc_env == 'prod':
-            manifests_path = module_target_sat.download_file(
-                file_url=settings.fake_manifest.url['default']
-            )[0]
             module_target_sat.cli.Subscription.upload(
-                {'file': manifests_path, 'organization-id': module_rhc_org.id}
+                {'file': module_entitlement_manifest.filename, 'organization-id': module_rhc_org.id}
```

Comment:
Ping @jameerpathan111 to assess any impact on RHC testing here !

Reply:
Ping @jameerpathan111 again !!!

Reply:
lgtm